test/integration/verify/controls/aws_db_parameter_groups_loop.rb

```diff
@@ -0,0 +1,24 @@
+title 'Test AWS RDS Parameter Groups in bulk'
+
+aws_rds_db_parameter_group_family   = attribute(:aws_rds_db_parameter_group_family, default: '', description: 'The name of the DB parameter group family that this DB parameter group is compatible with.')
+aws_rds_db_parameter_group_arn      = attribute(:aws_rds_db_parameter_group_arn, default: '', description: 'The Amazon Resource Name (ARN) for the DB parameter group.')
+
+control 'aws-db-parameter-groups-loop-1.0' do
+
+  impact 1.0
+  title 'Loop across AWS RDS Parameter Group resource using singular resource for detail.'
+
+  aws_db_parameter_groups.db_parameter_group_names.each do |parameter_group|
+    describe aws_db_parameter_group(db_parameter_group_name: parameter_group) do
+      it { should exist }
+    end
+  end
+
+  aws_db_parameter_groups.where { db_parameter_group_family == aws_rds_db_parameter_group_family }.db_parameter_group_names.each do |parameter_group|
+    describe aws_db_parameter_group(db_parameter_group_name: parameter_group) do
+      it { should exist }
+      its('db_parameter_group_family')      { should include aws_rds_db_parameter_group_family }
```

Comment:
am not sure if you have tested this, because in [filter table](https://github.com/inspec/inspec-aws/pull/342/files?file-filters%5B%5D=.rb&file-filters%5B%5D=.tf#diff-310e66c7420cce7a4a7007fbe96a8d7fdd8ecc3c5b3e625f0cede85a7c45d2a9R17-R20) you have defined the column in pluralised format and here you refer to singular version

Reply:
Corrected